### PR TITLE
feat(skilavottord): adding update mileage resolver

### PR DESCRIPTION
**Pull request by essgje**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/skilavottord/ws/src/app/modules/vehicle/vehicle.resolver.ts [hunk 1]
```diff
@@ -102,4 +102,13 @@ export class VehicleResolver {
     newVehicle.vehicleId = vehicle.permno
     return await this.vehicleService.create(newVehicle)
   }
+
+  @Mutation(() => Boolean)
+  async updateSkilavottordVehicleMileage(
+    @CurrentUser() user: User,
+    @Args('permno') permno: string,
+    @Args('permno') mileage: number,
```
Comment (birkirkristmunds) [inline apps/skilavottord/ws/src/app/modules/vehicle/vehicle.resolver.ts:110]: @Args('permno') mileage: number,

Ætti þetta ekki að vera 

 @Args('permno') permno: number,

**Review by birkirkristmunds — CHANGES_REQUESTED** [review]

File: apps/skilavottord/ws/src/app/modules/vehicle/vehicle.resolver.ts (hunk 1 above)
Comment (essgje) [reply apps/skilavottord/ws/src/app/modules/vehicle/vehicle.resolver.ts:110]: permno og mileage inn

**Review by birkirkristmunds — APPROVED** [review]
LGTM

**Review by birkirkristmunds — APPROVED** [review]
LGTM

**Pull request merged**